### PR TITLE
refactor(system-users) change config for system-user 

### PR DESCRIPTION
**Pull request by TsaghikKhachatryan**
## Purpose
Secure setup of system users by default

## Approach
system user name on a configuration variable. A default fallback value might be set.
system user password on a configuration variable. NO default fallback value should be set.

### Changes checklist

>   system-user:
>     username: ${SYSTEM_USER_NAME:some-system-user-name}
>     password: ${SYSTEM_USER_PASSWORD}

### Learning
https://issues.folio.org/browse/MSEARCH-608

### CODE REVIEW

**Review by viacheslavkol — COMMENTED** [review]
You also need to remove "SYSTEM_USER_PASSWORD" from ModuleDescriptor

**Review by psmagin — CHANGES_REQUESTED** [review]
please update README and NEWS

File: NEWS.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 * Description ([ISSUE_NUMBER](https://issues.folio.org/browse/ISSUE_NUMBER))
 
 ### Bug fixes
-* Description ([ISSUE_NUMBER](https://issues.folio.org/browse/ISSUE_NUMBER))
+* Fix secure setup of system users by default ([MSEARCH-608](https://issues.folio.org/browse/MSEARCH-608))
```
Comment (TsaghikKhachatryan) [inline NEWS.md:13]: @psmagin is it correct to have this update under Bug fixes?

Comment (psmagin) [reply NEWS.md:13]: It's okay

**Review by psmagin — APPROVED** [review]

**Review by mukhiddin-yusuf — APPROVED** [review]

**Pull request merged**